esx_service/utils/auth_api.py

```diff
@@ -22,42 +22,68 @@
 import error_code
 import log_config
 import logging
+from error_code import ErrorCode
+from error_code import ErrorInfo
+
+def generate_error_info_for_internal_error(secondary_err_msg):
```

Comment:
Python does support variable argument with special syntax *args and **kwargs in function definitions.